vulnerable_people_form/form_pages/not_eligible_england.py

```diff
@@ -6,5 +6,5 @@
 @form.route("/not-eligible-england", methods=["GET"])
 def get_not_eligible_england():
     return render_template_with_title(
-        "not-eligible-england.html", back_url=dynamic_back_url()
+        "not-eligible-england.html", previous_path=dynamic_back_url()
```

Comment:
does this fix this https://trello.com/c/u07YqQJo/280-sorry-youre-not-eligible-for-the-service-update-back-link?

Reply:
Yes 🙂